clang/test/compiler-course/chastov_v_lab1/chastov_v_lab1.cpp

```diff
@@ -0,0 +1,61 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/DataTypes_Chastov_Vyacheslav_FIIT2_ClangAST%pluginext -plugin DataTypes_Chastov_Vyacheslav_FIIT2_ClangAST %s -fsyntax-only 2>&1 | FileCheck %s
+
+// CHECK: Base1
+// CHECK-NEXT: |_Fields
+// CHECK-NEXT: | |_ (no fields)
+// CHECK-NEXT: |_Methods
+// CHECK-NEXT: | |_ virtualMethod (void()|public|virtual|pure)
+
+// CHECK: Base2
+// CHECK-NEXT: |_Fields
+// CHECK-NEXT: | |_ (no fields)
+// CHECK-NEXT: |_Methods
+// CHECK-NEXT: | |_ baseMethod (void()|public|virtual)

```

Comment:
```suggestion
// CHECK-LABEL: Base1
// CHECK-NEXT: |_Fields
// CHECK-NEXT: | |_ (no fields)
// CHECK-NEXT: |_Methods
// CHECK-NEXT: | |_ virtualMethod (void()|public|virtual|pure)

// CHECK-LABEL: Base2
// CHECK-NEXT: |_Fields
// CHECK-NEXT: | |_ (no fields)
// CHECK-NEXT: |_Methods
// CHECK-NEXT: | |_ baseMethod (void()|public|virtual)
```
Replace `CHECK` -> `CHECK-LABEL`. Also in other tests

Reply:
fixed